cartoframes/data/dataset/registry/dataframe_dataset.py

```diff
@@ -32,7 +32,7 @@ def create(cls, data, credentials=None, schema=None):
 
         save_index_as_column(data)
 
-        return cls(data)
+        return cls(data, credentials, schema)
```

Comment:
I think it should be enough with `cls(data, credentials)` because I think `schema` is not used.